examples/rbac/nginx/nginx-ingress-controller-rbac.yml

```diff
@@ -69,6 +70,7 @@ rules:
       - secrets
     verbs:
       - get
+      - update
```

Comment:
you only need access to update a single config map, the one used for the leader election, so that should be in a separate rule, with a `resourceNames` stanza listing the particular name of the config map you need to update